src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -168,24 +168,26 @@
 
 helps['acr repository delete'] = """
     type: command
-    short-summary: Delete a repository, manifest, or tag in a container registry.
-    parameters:
-        - name: --manifest
-          populator-commands:
-            - az acr repository show-manifests
+    short-summary: Delete a repository or image in a container registry.
     examples:
         - name: Delete a repository from a container registry.
           text:
-            az acr repository delete -n MyRegistry --repository MyRepository
-        - name: Delete a tag from a repository. This does not delete the manifest referenced by the tag or any associated layer data.
+            az acr repository delete -n MyRegistry --repository hello-world
+        - name: Delete an image by tag. This deletes the manifest referenced by 'hello-world:latest', all other tags referencing the manifest, and any associated layer data.
           text:
-            az acr repository delete -n MyRegistry --repository MyRepository --tag MyTag
-        - name: Delete the manifest referenced by a tag. This also deletes any associated layer data and all other tags referencing the manifest.
+            az acr repository delete -n MyRegistry --image hello-world:latest
+        - name: Delete an image by sha256 based manifest digest. This deletes all tags referencing the manifest and any associated layer data.
           text:
-            az acr repository delete -n MyRegistry --repository MyRepository --tag MyTag --manifest
-        - name: Delete a manifest using a sha256 based digest. This also deletes any associated layer data and all tags referencing the manifest.
+            az acr repository delete -n MyRegistry --image hello-world@sha256:abc123
+"""
+
+helps['acr repository untag'] = """
+    type: command
+    short-summary: Untag an image in a container registry.
+    examples:
+        - name: Untag an image from a repository. This does not delete the manifest referenced by the tag or any associated layer data.
```

Comment:
The text "This does not delete the manifest..." should be moved into the `long-summary` for this command vice the example text.